src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV2.scala

```diff
@@ -80,8 +80,10 @@ trait UserRoutesV2 extends SamUserDirectives with SamRequestContextDirectives wi
                 val samRequestContext = samRequestContextWithoutUser.copy(samUser = Some(samUser))
                 pathEndOrSingleSlash {
                   get {
-                    complete {
-                      userService.getSamUserCombinedState(samUser, samRequestContext, resourceService)
+                    parameters("groupsContributingToMostMembershipsLimit".as[Int].?) { topGroupsLimit =>
+                      complete {
+                        userService.getSamUserCombinedState(samUser, topGroupsLimit.getOrElse(0), samRequestContext, resourceService)
+                      }
```

Comment:
nit:
`getSamUserCombinedState(WorkbenchEmail, …)` has its `topGroupsLimit` argument fourth
`getSamUserCombinedState(SamUser, …)` has its `topGroupsLimit` second

this makes the invocations in `AdminRoutes` and `UserRoutesV2` look more different than they are; reorder arguments?